protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -275,6 +276,7 @@ func buildEtcdBackupManagerContainer(c *EtcdCluster) *v1.Container {
 		command = append(command, "--client-ca-file", c.TLSCA)
 		command = append(command, "--client-cert-file", c.TLSCert)
 		command = append(command, "--client-key-file", c.TLSKey)
+		command = append(command, "--listen-metric-urls", "https://__address__:"+strconv.Itoa(c.MetricsPort))
```

Comment:
I noticed that this section of the code will only be applied if the spec etcdBackup is fufilled. Is this intended or should the listen-metric-urls be set on the main etcd node ?
Also, to alleviate the etcd 3.2 compatibility, we could use environment variable instead of command line option.